### DISCUSSION TITLE
Fleet 'Add New Host' should point to Downloads section

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2340] [Ideas] [answered]
When viewing the Fleet manager page and clicking 'Add New Host' it should point to the 'Downloads' page where the pre-configured scripts are kept.  Or at least it should have verbiage that instructs the person to look there.  It seems a bit disjointed the way it is.

Or perhaps it would be better if the pre-configured **soquery** scripts were located in a 'Downloads' section directly on the Fleet manager page instead of the SO sidebar.  That seems more logical.

I love SO and it's getting better every day, but some of this stuff seems like it gets overlooked because the creative team understands all the pieces so well that they sometimes forget that the users don't always understand how everything ties together. Especially with a major interface change.  Keep up the good work guys!  I'm enrolling in the paid training course now.

**Comment by TOoSmOotH** [comment, marked as answer]
The main issue here is Fleet is completely separate from SOC. Downloads are in SOC and we put the files there for convenience. This would take development effort by the Fleet devs to detect if it is running in a Security Onion grid and adjust the link it provides. They are busy with FleetDM but you could ask over there. 

Our focus at the moment is fixing bugs and priority features that will improve the overall user experience in managing your grid and finding evil. I wouldn't say we "overlooked" a feature like this as the Fleet interface is out of our control. Please keep the feedback coming so we can improve the overall experience.
